narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -233,6 +234,8 @@ function get_hotfix_zip_list_file {
 
 	local is_new_file
 
+	trap 'remove_txt_file "${zip_list_file}"' EXIT 
```

Comment:
The $zip_list_file should not be removed but used whenever the script runs. If it is too old, it is generated from scratch; otherwise, it is used as is. There is actually a switch to control that.